packages/mask/background/services/wallet/services/wallet/index.ts

```diff
@@ -308,6 +360,26 @@ export function createWalletFromPrivateKey(name: string, privateKey: string) {
     return addWalletFromPrivateKey(ImportSource.LocalGenerated, name, privateKey)
 }
 
+export async function generateWalletFromPrivateKey(privateKey: string) {
```

Comment:
Align `generateAddressFromMnemonicWords`:

```suggestion
export async function generateAddressFromPrivateKey(privateKey: string) {
```